src/nspawn/nspawn.c

```diff
@@ -374,7 +373,7 @@ static int parse_argv(int argc, char *argv[]) {
                 { "overlay-ro",            required_argument, NULL, ARG_OVERLAY_RO        },
                 { "machine",               required_argument, NULL, 'M'                   },
                 { "slice",                 required_argument, NULL, 'S'                   },
-                { "setenv",                required_argument, NULL, ARG_SETENV            },
+                { "setenv",                required_argument, NULL, 'E'                   },
```

Comment:
it probably makes sense to add the same to src/run/run.c and src/machine/machinectl.c. They all have an equivalent switch `--setenv=`